tests/Builder/ShowBuilderTest.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Builder;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use PHPUnit\Framework\TestCase;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Admin\FieldDescriptionCollection;
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+use Sonata\AdminBundle\Builder\ShowBuilderInterface;
+use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
+use Sonata\DoctrineORMAdminBundle\Builder\ShowBuilder;
+use Sonata\DoctrineORMAdminBundle\Model\ModelManager;
+use Symfony\Component\Form\Guess\TypeGuess;
+
+/**
+ * @author Marko Kunic <kunicmarko20@gmail.com>
+ */
+class ShowBuilderTest extends TestCase
+{
+    /**
+     * @var TypeGuesserInterface
+     */
+    private $guesser;
+
+    /**
+     * @var ShowBuilderInterface
+     */
+    private $showBuilder;
+
+    public function setUp()
+    {
+        $this->guesser = $this->prophesize(TypeGuesserInterface::class);
+
+        $this->showBuilder = new ShowBuilder(
+            $this->guesser->reveal(),
+            ['fakeTemplate' => 'fake']
+        );
+    }
+
+    public function testGetBaseList()
+    {
+        $this->assertInstanceOf(FieldDescriptionCollection::class, $this->showBuilder->getBaseList());
+    }
+
+    public function testAddFieldNoType()
+    {
+        $fieldDescriptionCollection = $this->prophesize(FieldDescriptionCollection::class);
+        $fieldDescription = $this->prophesize(FieldDescriptionInterface::class);
+        $admin = $this->prophesize(AdminInterface::class);
+        $typeGuess = $this->prophesize(TypeGuess::class);
+        $modelManager = $this->prophesize(ModelManager::class);
+        $typeGuess = $this->prophesize(TypeGuess::class);
```

Comment:
duplicated line with 60

Reply:
nice catch!

Reply:
I am looking at what scrutinizer says, there are a few errors out there, can you check it?

Reply:
errors on `AuditReaderTest` do not make sense?

Reply:
It is because of docblocks declaring types